.travis.yml

```diff
@@ -4,11 +4,11 @@ cache: pip
 services:
   - docker
 env:
-  - ANSIBLE=2.2.3
-  - ANSIBLE=2.3.3
-  - ANSIBLE=2.4.3
+  - ANSIBLE='>2.2.0,<2.3.0'
```

Comment:
Can you change it to `>=` instead of `>` ?

Reply:
Sure. I did initially think that it might make sense to.